docs/deployment/helm.md

```diff
@@ -108,6 +108,7 @@ We have introduced the following Chart parameters.
 | Name                        | Type    | Default                                 | description                                                                                                                              |
 |-----------------------------|---------|-----------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|
 | `master.*`                  | dict    |                                         | NFD master deployment configuration                                                                                                      |
+| `master.port` | integer | | Specifies the TCP port that nfd-master listens for incoming requests. |
```

Comment:
I know it renders properly but this is the only line which is not aligned :( otherwise lgtm